feat: Add HACBS QE team to IC rotation.

rhopp opened the pull request:
# Description

This PR is adding (hopefully) all Stonesoup QEs to interrupt catcher rotation.

Complete list of people (and their order) is as follows:
```
[
        {"username": "jinqi", "id": "U035JT3C9TP"},
        {"username": "jkopriva", "id": "UJNFUKWN8"},
        {"username": "Roman Jeczkowiak", "id": "U029ED7DB1D"},
        {"username": "msawood", "id": "UUQJT4YVA"},
        {"username": "chuo", "id": "UFGRUV2B1"},
        {"username": "Tríona Doyle", "id": "U0167DWFQEM"},
        {"username": "Priti", "id": "UMTPY71PD"},
        {"username": "rhopp", "id": "UGZCNQU69"},
        {"username": "praveen_thangadancha", "id": "UJ5T3GBMY"},
        {"username": "sselvan", "id": "U019U19J872"},
        {"username": "Flavius Lacatusu", "id": "US71VQW3C"},
        {"username": "Jitendar Singh", "id": "UMSJFPB37"},
        {"username": "Kasem Alem", "id": "U02NUAB5X3M"},
        {"username": "Sushanta Das", "id": "U02CNJ70PB7"},
        {"username": "Pavel Sturc", "id": "UKYR3KDDX"},
        {"username": "Alessandro Babrarossa", "id": "U03A1NFTA4R"},
        {"username": "nshprai", "id": "U02H15M8H25"},
        {"username": "xjiang", "id": "UM44FF4KG"},
        {"username": "Rafaela Soares", "id": "U04D9K8A36J"}
]
```
(you can check this by base64 decoding the `cm_people_list.yaml` config map.

Dannyb48 commented:
We are missing `Dheeraj`

flacatus commented:
Is         {"username": "Roman Jeczkowiak", "id": "U029ED7DB1D"}, from stonesoup? 😕

rhopp commented:
I've updated the list based on feedback.
* Removed Praveen
* Added Dheeraj, Anand and Borja
new list looks like this:
```
[
        {"username": "jinqi", "id": "U035JT3C9TP"},
        {"username": "jkopriva", "id": "UJNFUKWN8"},
        {"username": "Roman Jeczkowiak", "id": "U029ED7DB1D"},
        {"username": "msawood", "id": "UUQJT4YVA"},
        {"username": "chuo", "id": "UFGRUV2B1"},
        {"username": "Tríona Doyle", "id": "U0167DWFQEM"},
        {"username": "Priti", "id": "UMTPY71PD"},
        {"username": "rhopp", "id": "UGZCNQU69"},
        {"username": "Dheeraj", "id": "U036P0K0W5D"},
        {"username": "ansingh", "id": "UUC8YC1FU"},
        {"username": "sselvan", "id": "U019U19J872"},
        {"username": "Flavius Lacatusu", "id": "US71VQW3C"},
        {"username": "Jitendar Singh", "id": "UMSJFPB37"},
        {"username": "Kasem Alem", "id": "U02NUAB5X3M"},
        {"username": "Sushanta Das", "id": "U02CNJ70PB7"},
        {"username": "Borja Luengo Prieto", "id": "U02A7PMS5DF"},
        {"username": "Pavel Sturc", "id": "UKYR3KDDX"},
        {"username": "Alessandro Babrarossa", "id": "U03A1NFTA4R"},
        {"username": "nshprai", "id": "U02H15M8H25"},
        {"username": "xjiang", "id": "UM44FF4KG"},
        {"username": "Rafaela Soares", "id": "U04D9K8A36J"}
]
```

Dannyb48 reviewed (approved):
LGTM

flacatus commented:
Are all this people aware that will be included in this list? Im not sure about gitops qe for example

rhopp commented:
/retest

rhopp commented:
/retest

rhopp commented:
/retest
